src/routes/expense.router.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const express = require('express');
+
+const { expenseController } = require('../controllers/expense.controller.js');
```

Comment:
Consider using `import` instead of `require` for importing modules to follow the ES6 module syntax. Example: import { expenseController } from '../controllers/expense.controller.js';